pkg/vadmin/restart_node_vc.go

```diff
@@ -46,6 +47,11 @@ func (v *VClusterOps) RestartNode(ctx context.Context, opts ...restartnode.Optio
 	vcOpts := v.genStartNodeOptions(&s, certs)
 	err = v.VStartNodes(vcOpts)
 	if err != nil {
+		reIPNoClusterQuorumError := &vops.ReIPNoClusterQuorumError{}
```

Comment:
I think we should log a message here to say we got the error. This is usually done in `vcErrors.LogFailure`. Can we move this logic into there?